src/database/user_operations.py

```
@@ -1,144 +1,134 @@
             if row:
                 uname = row[0]
-                ah = hashlib.sha1(f"{uname}:{tmp}".encode()).hexdigest().upper()
-                await cur.execute("UPDATE account SET sha_pass_hash=%s WHERE email=%s", (ah, mu))
-                logger.info(f"Пароль успешно сброшен для e-mail: {mu}")
-            else:
-                logger.warning(f"Не удалось сбросить пароль: e-mail {mu} не найден в таблице account")
-    
-    if not affected:
-        logger.warning(f"Не удалось сбросить пароль: e-mail {mu} не найден в battlenet_accounts")
-        return None
+                account_hash = hashlib.sha1(f"{uname}:{tmp}".encode()).hexdigest().upper()
```

Ruff (0.12.2)

Probable use of insecure hash functions in `hashlib`: `sha1`

(S324)